faq/apple-silicon-m1-as-a-service.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: Apple silicon M1 as-a-service
-  description: Let us address the most common questions you have about Apple silicon M1 as-a-service.
+  description: Discover how to use M1 for applications' development.
```

Comment:
`Discover how to use M1 for application development` would sound better I think?